docs/reference/builder.md

```diff
@@ -70,9 +73,9 @@ to be created - so `RUN cd /tmp` will not have any effect on the next
 instructions.
 
 Whenever possible, Docker will re-use the intermediate images,
-accelerating `docker build` significantly (indicated by `Using cache` -
-see the [`Dockerfile` Best Practices
-guide](/articles/dockerfile_best-practices/#build-cache) for more information):
+accelerating `docker build` process significantly. It is indicated by
+the `Using cache` message in the console output.
+(For more information, see the [Build cache](/articles/dockerfile_best-practices/#build-cache)):
```

Comment:
This sentence feels incomplete ("see the Build cache"). Perhaps "see the Build cache section in the `Dockerfile` best practices guide", making "Build cache section" a link